src/app/schedule-view/component.html

```diff
@@ -8,6 +8,13 @@
         <span class="schedule-switch" (click) = "nextSchedule()">&#9654;</span>
       </div>
       <button class="schedule-button" (click) = "clear()">Clear</button>
+      <!-- <button class="schedule-button">Download ICS</button>
+      <button class="schedule-button" [disabled]="isTemporary">Copy Schedule Link</button> -->
+      <button class="schedule-button" (click) = "showExport()">Export as ...</button>
+      <button *ngIf="isExporting" class="schedule-button" [disabled]="!isExporting" (click) = "downloadImage()">Image</button>
+      <button *ngIf="isExporting" class="schedule-button" [disabled]="!isExporting">iCalendar</button>
```

Comment:
Can we remove these since they aren't implemented yet?

Reply:
I will.